web/app/js/components/util/MetricUtils.jsx

```diff
@@ -112,8 +112,12 @@ const processStatTable = table => {
   let rows = _compact(table.podGroup.rows.map(row => {
     let runningPodCount = parseInt(row.runningPodCount, 10);
     let meshedPodCount = parseInt(row.meshedPodCount, 10);
+    let rowKey = `${row.resource.namespace}-${row.resource.type}-${row.resource.name}`;
+    if (row.resource.type === "trafficsplit" && row.tsStats) {
```

Comment:
if only one of these two things is true, is that expected? or a bug?

Reply:
You're right, `row.resource.type === "trafficsplit"` and `row.tsStats` would only occur together so I can remove one of the checks.